lib/pummpcomm/session/exchange/read_cgm_page.ex

```diff
@@ -1,14 +1,31 @@
 defmodule Pummpcomm.Session.Exchange.ReadCgmPage do
-  alias Pummpcomm.Session.Command
-  alias Pummpcomm.Session.Response
+  @moduledoc """
+  Reads a (past) CGM page.  To cread the current CGM page, use `Pummpcomm.Session.Exchange.GetCurrentCgmPage`.
```

Comment:
See other comment on GetCurrentCgmPage. This is pretty close documentation, but not quite 100% accurate.

A little more trivia, cgm pages count up, so page 1 comes before page 2 and the number climbs as new pages are written. That why GetCurrentCgmPage has to exist, so we can fetch the current page's index. History pages are the opposite. Getting history page 0 is always the most recent, and to fetch older pages you fetch 1 then 2, etc. So for history, there's no corresponding GetCurrentHistoryPage call.